### PR TITLE
Add XCUI Test

### PR DESCRIPTION
**Pull request by macsleven**
[XCUI Documentation](https://developer.apple.com/documentation/xctest/user_interface_tests)

- [x] Create and delete wallet: tests creating and deleting of wallets. It runs twice, first with password then PIN.

- [x] Restore Wallet: test wallet restore and seed inputs using a constant seed phrase. It runs twice, first with password then PIN

- [x] Test seed backup: create a new wallet, attempt to get seed from the screen and verify the seed.

### CODE REVIEW

**Comment by JustinBeBoy** [comment]
@macsleven please resolve the conflict

File: Decred WalletUITests/Decred_WalletUITests.swift [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+    
+    func testCreateNewWalletWithPIN() throws {
+        let app = XCUIApplication()
+        app.launchArguments.append("--UITests")
+        app.launch()
+        
+        let permission = app.alerts["“Decred Wallet Testnet” Would Like to Send You Notifications"].scrollViews.otherElements.buttons["Allow"]
```
Comment (JustinBeBoy) [inline Decred WalletUITests/Decred_WalletUITests.swift:31]: It cann't handle when the alert notification shows in the first time

File: Decred WalletUITests/RestoreExistingWalletUITest.swift [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+        }
+        
+        let restoreExistingWallet = app.buttons.element(matching: .button, identifier: "restoreExistingWallet")
+        if restoreExistingWallet.waitForExistence(timeout: 5) {
+            restoreExistingWallet.tap()
+        }
+        
+        sleep(1)
+        
+        self.seedUIInput(app: app)
```
Comment (JustinBeBoy) [inline Decred WalletUITests/RestoreExistingWalletUITest.swift:45]: false when keyboard is not in English

File: Decred WalletUITests/Decred_WalletUITests.swift [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+    
+    
+   /* func testLaunchPerformance() throws {
+        if #available(macOS 10.15, iOS 13.0, tvOS 13.0, *) {
+            // This measures how long it takes to launch your application.
+            measure(metrics: [XCTOSSignpostMetric.applicationLaunch]) {
+                XCUIApplication().launch()
+            }
+        }
+    }*/
```
Comment (JustinBeBoy) [inline Decred WalletUITests/Decred_WalletUITests.swift:225]: is it necessary ?

**Review by JustinBeBoy — CHANGES_REQUESTED** [review]
I left a few comments that should be addressed

File: Decred WalletUITests/RestoreExistingWalletUITest.swift (hunk 2 above)
Comment (macsleven) [reply Decred WalletUITests/RestoreExistingWalletUITest.swift:45]: this is a tests the default language and hard coded strings are in english.

**Review by JustinBeBoy — APPROVED** [review]

**Review by C-ollins — APPROVED** [review]

**Pull request merged**